src/picongpu/include/plugins/radiation/utilities.hpp

```diff
@@ -70,9 +70,17 @@ namespace util
         }
     };
 
-}
-
-
-
-
-
+  /** power function
+    *
+    * @tparam T_Type - base type
+    * @param x - base value
+    * @param exp - exponent
+    * @return std::pow(x, exp)
+    */
+  template< typename T_Type >
+  constexpr HDINLINE T_Type pow( T_Type x , uint32_t exp )
+  {
+    return exp == 0 ? 1 : util::pow( x, exp - 1 );
```

Comment:
please update to:

```C++
return exp == 0 ? 1 : (
    exp == 1 ? x : util::pow( x, exp - 1 )
);
```

This save one multiplication. An effect of this change is only visible in compute intensive use cases.